composables/useIdentity.ts

```diff
@@ -31,19 +31,19 @@ export default function useIdentity({
   )
 
   const identityApi = computed(() => apiInstanceByPrefix(identityPrefix.value))
-
-  const identity = computed<IdentityFields>(
-    () => data.value?.value.identity || {}
-  )
+  const clientName = computed(() => (isDotAddress.value ? 'pid' : 'kid'))
 
   const { data, refetch, loading } = useGraphql({
-    clientName: computed(() => (isDotAddress.value ? 'pid' : 'kid')),
```

Comment:
The clientName should be a computed reactive value instead of a specific prefix, or the identity data would not be updated while swtiching network.

Reply:
aahh, I see noted. Will update it 🙏 @Jarsen136 @roiLeo

Reply:
updated in 186a197f406cf18ca230cf1a9ed5a78eb1ca8d8f